### ISSUE TITLE
build error

### ISSUE THREAD

**Issue by mark-fangzhou-xie** [timholy/ProfileView.jl#115]
When I was installing this package, I found the following error:

```
(v1.1) pkg> build ProfileView
  Building LibCURL ─→ `~/.julia/packages/LibCURL/lWJxD/deps/build.log`
  Building WinRPM ──→ `~/.julia/packages/WinRPM/Y9QdZ/deps/build.log`
  Building Homebrew → `~/.julia/packages/Homebrew/s09IX/deps/build.log`
  Building Cairo ───→ `~/.julia/packages/Cairo/p68X8/deps/build.log`
┌ Error: Error building `Cairo`: 
│ WARNING: redefining constant DEBIAN_VERSION_REGEX
│ WARNING: redefining constant have_sonames
│ WARNING: redefining constant sonames
│ WARNING: redefining constant EXTENSIONS
│ Error: glib: undefined method `uses_from_macos' for Formulary::FormulaNamespace0c4af498993b020c6edb695840c0ca4f::Glib:Class
│ Error: glib: undefined method `uses_from_macos' for Formulary::FormulaNamespace0c4af498993b020c6edb695840c0ca4f::Glib:Class
│ 
│ signal (11): Segmentation fault: 11
│ in expression starting at /Users/xiefangzhou/.julia/packages/Cairo/p68X8/deps/build.jl:176
│ _platform_strcmp at /usr/lib/system/libsystem_platform.dylib (unknown line)
│ Allocations: 19575723 (Pool: 19572904; Big: 2819); GC: 43
└ @ Pkg.Operations /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.1/Pkg/src/Operations.jl:1075
  Building Gtk ─────→ `~/.julia/packages/Gtk/aP55V/deps/build.log`
┌ Error: Error building `Gtk`: 
│ Error: glib: undefined method `uses_from_macos' for Formulary::FormulaNamespace0c4af498993b020c6edb695840c0ca4f::Glib:Class
│ ERROR: LoadError: ArgumentError: `brew info` failed for ["cairo", "gdk-pixbuf", "glib", "libcroco", "pango"]!
│ Stacktrace:
│  [1] json(::Array{String,1}) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:192
│  [2] info(::Array{String,1}) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:229
│  [3] #direct_deps#10(::Bool, ::Function, ::String) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:285
│  [4] #direct_deps at ./tuple.jl:0 [inlined]
│  [5] #direct_deps#15(::Bool, ::Function, ::Homebrew.BrewPkg) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:289
│  [6] #direct_deps at ./tuple.jl:0 [inlined]
│  [7] #deps_tree#17(::Bool, ::Function, ::String) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:315
│  [8] #deps_tree at ./tuple.jl:0 [inlined]
│  [9] #deps_sorted#24(::Bool, ::Function, ::String) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:379
│  [10] deps_sorted at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:379 [inlined]
│  [11] #add#29(::Bool, ::Bool, ::Function, ::String) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:420
│  [12] add(::String) at /Users/xiefangzhou/.julia/packages/Homebrew/s09IX/src/API.jl:412
│  [13] top-level scope at /Users/xiefangzhou/.julia/packages/Gtk/aP55V/deps/build.jl:60
│  [14] include at ./boot.jl:326 [inlined]
│  [15] include_relative(::Module, ::String) at ./loading.jl:1038
│  [16] include(::Module, ::String) at ./sysimg.jl:29
│  [17] include(::String) at ./client.jl:403
│  [18] top-level scope at none:0
│ in expression starting at /Users/xiefangzhou/.julia/packages/Gtk/aP55V/deps/build.jl:43
└ @ Pkg.Operations /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.1/Pkg/src/Operations.jl:1075
```
And the following system information if helpful:
OS: macos 10.14.5
julia: 
```
julia> versioninfo()
Julia Version 1.1.1
Commit 55e36cc (2019-05-16 04:10 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin15.6.0)
  CPU: Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
```

I am not sure what is wrong here. Thank you for your help!

**Comment by timholy** [comment]
Duplicate of #112, #110.

**Comment by timholy** [comment]
Should be fixed if you use Julia 1.3 and up-to-date packages
